tests/providers/google/cloud/operators/test_dataproc.py

```diff
@@ -104,7 +104,7 @@
         "autoscaling_config": {"policy_uri": "autoscaling_policy"},
         "config_bucket": "storage_bucket",
         "initialization_actions": [
-            {"executable_file": "init_actions_uris", "execution_timeout": "600s"}
+            {"executable_file": "init_actions_uris", "execution_timeout": "600.0s"}
```

Comment:
This is my only question -- does DataProc accept this, or does it only accept whole-integer seconds here.

Reply:
Should be int:
https://developers.google.com/protocol-buffers/docs/reference/google.protobuf?&_ga=2.173161123.14377043.1581598687-968688242.1573564810#duration

Reply:
https://developers.google.com/protocol-buffers/docs/proto3#json

@saguziel WDYT?

Reply:
the python typing indicates float, but I think that is just within the airflow project.

I will just cast it to int since that is guaranteed to not break anything